core/health/rest/src/main/resources/openapi/health-check-api.yaml

```diff
@@ -39,6 +39,13 @@ paths:
             type: integer
             format: int32
             default: 5000
+        - name: tag
```

Comment:
Does swagger work on Minion without any changes ?
From my understanding this only works on OpenNMS but the URL here in this file is hardcoded to Minion URL

Reply:
This is only the doc. We don't have embedded Swagger UI related to this. User will need to input URL by themself when they use it. The URL we put here is just an example.

Reply:
Opened https://issues.opennms.org/browse/NMS-13386 to address docs.